clients/bcexporter/config/Config.py

```diff
@@ -15,5 +14,5 @@ def _get_config(self):
             return yaml.safe_load(stream)
 
     def _get_chains(self):
-        with open('./config/chains.json', "r") as stream:
+        with open("./config/chains.json", "r") as stream:
```

Comment:
Is chains.json mounted yet in docker-compose? It should not be expected that chains.json is part of the python build, given it's a config file that should be editable and changeable